libclc/CMakeLists.txt

```diff
@@ -347,7 +342,7 @@ foreach( t ${LIBCLC_TARGETS_TO_BUILD} )
   endif()
 
   if( IS_NATIVE_CPU_ARCH AND OS STREQUAL linux)
-    LIST( APPEND dirs native_cpu-unknown-linux )
+    LIST( APPEND clc_dirs native_cpu-unknown-linux )
```

Comment:
This is appending to `clc_dirs` but then `set( clc_dirs`  comes later, which will overwrite this line. Sadly I spy a bug in the upstream code where `set( clc_dirs ${dirs} generic` won't pick up on `amdgpu` (above). Luckily it's not currently a problem as there's no `libclc/clc/amdgpu` but I'll need to fix that soon.

Regardless, I think we'll need a new `libspirv_dirs` variable. There's no `libclc/clc/native_cpu-unknown-linux` so it doesn't make sense in `clc_dirs`. But there's also no `libclc/native_cpu-unknown-linux` so `opencl_dirs` doesn't make sense either.

I think `libspirv_dirs` can be the same as `opencl_dirs` but with `native_cpu-unknown-linux`.